checksum.go

```
@@ -316,6 +316,38 @@ func (c ChecksumType) String() string {
+		return ChecksumXXHash64.Hasher(), obj.ChecksumXXHash64
+	case "XXHASH3":
+		return ChecksumXXHash3.Hasher(), obj.ChecksumXXHash3
+	case "XXHASH128":
+		return ChecksumXXHash128.Hasher(), obj.ChecksumXXHash128
+	case "":
+		return nil, ""
+	}
+	return nil, ""
+}
+
+
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofumpt)